src/internal/DismissableContainer/index.tsx

```diff
@@ -86,6 +86,8 @@ export const DismissableContainer = withTheme(
 					{children}
 					{onDismiss && (
 						<DismissButton
+							title="Dismiss Alert"
```

Comment:
This is not always an alert, it can be the onboarding steps, or something else. It's better if each component that uses the `DismissableContainer` pass their own title.

Reply:
Ahh, I missed this, I'll update this.

Reply:
I've added a default title of 'Dismiss' if nothing is passed.